tee-worker/litentry/primitives/src/validation_data.rs

```diff
@@ -34,10 +34,16 @@ pub enum TwitterValidationData {
 
 #[derive(Encode, Decode, Clone, Debug, PartialEq, Eq, TypeInfo, MaxEncodedLen)]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
-pub struct DiscordValidationData {
-	pub channel_id: ValidationString,
-	pub message_id: ValidationString,
-	pub guild_id: ValidationString,
+pub enum DiscordValidationData {
+	DiscordPublicMessage {
```

Comment:
Shall we have a consitent naming with the Twitter?
I mean `DiscordPublicMessage` vs `PublicTweet`

Reply:
on a second thought, I think `Discord*` is redundant as it's already in `DiscordValidationData`. I'll change it back to just `PublicMessage`